File: 1 - INICIANTE/C++17/1131.cpp
#include <iostream>
#include <iomanip>
using namespace std;

int main() {

int x,y,z,est1,est2,est3,contador;
est1 = 0;
est2 = 0;
est3 = 0;
contador = 0;

do {
cin >> x >> y;
contador++;
if (x>y) {
  est1++;
}
else if (y>x) {
  est2++;
}
else {
  est3++;
}

cout << "Novo grenal (1-sim 2-nao)" << endl;
cin >> z;
}
while (z!=2);

cout << contador << " grenais" << endl;
cout << "Inter:" << est1 << endl;
cout << "Gremio:" << est2 << endl;
cout << "Empates:" << est3 << endl;



if (est1>est2) {
  cout << "Inter venceu mais" << endl;
}
else if (est2>est1) {
  cout << "Gremio venceu mais" << endl;
}
else {
  cout << "Nao houve vencedor" << endl;
}

return 0; 
}

File: 1 - INICIANTE/python/1131.py
def leitura_dados(vet):
    dados = str(input()).split()
    vet[0] += 1
    #cont += 1
    if dados[0] > dados[1]:
        vet[1] += 1
        #inter += 1
    elif dados[0] < dados[1]:
        vet[2] += 1
        #gremio += 1
    else:
        vet[3] += 1


vet = [0, 0, 0, 0]
while True:
    leitura_dados(vet)
    while True:
        n = int(input(('Novo grenal (1-sim 2-nao)\n')))
        if n == 1 or n == 2:
            break
        else:
            continue
    if n == 2:
        print('''{} grenais
Inter:{}
Gremio:{}
Empates:{}'''.format(vet[0], vet[1], vet[2], vet[3]))
        if vet[1] > vet[2]:
            print('Inter venceu mais')
        elif vet[1] < vet[2]:
            print('Gemio venceu mais')
        else:
            print('Nao houve vencedor')
        break
